elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/MetaDataStore.java

```diff
@@ -79,6 +79,19 @@ public void populateEntityDictionary(EntityDictionary dictionary) {
                 .forEach(cls -> dictionary.bindEntity(cls, Collections.singleton(Join.class)));
     }
 
+    /**
+     * Bind dynamic compiled classes to dictionary.
+     * @param dynamicCompiledClasses Dynamically compiled classes
+     */
+    public void populateEntityDictionary(Set<Class<?>> dynamicCompiledClasses) {
```

Comment:
Would be nice if this was done in the constructor somehow (along with the other entities that are being populated).